src/overlays/actors/ovl_En_G_Switch/z_en_g_switch.c

```diff
@@ -80,6 +80,8 @@ void EnGSwitch_Init(Actor* thisx, PlayState* play) {
     s32 pad;
     EnGSwitch* this = (EnGSwitch*)thisx;
 
+    if (play) {}
+
     this->type = (this->actor.params >> 0xC) & 0xF;
```

Comment:
does this match with a play2 ( a fake match for recasting the input parameter)
For a while now we've known that they very likely originally passed GameState* to the 4 main actor function and made a temp to recast them in all cases. But we've yet to figure out if it works in ever single case.
If it does, I would prefer a `play2` for now over this

Reply:
It has regalloc issues unfortunately, there's probably a way to get it to work but I wasn't able to figure it out

Reply:
thanks for checking